docs/data/variant-calls.md

```diff
@@ -76,6 +86,16 @@ Data files containing the simple insertions and deletions (InDel) data called fr
 | ---------------- | --------------------------------- | --------------- | --------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
 | \*.indel.vcf.gz  | InDel results in VCF file format. | variant_calling | Simple Nucleotide Variation | <ul><li>Sanger WGS Variant Calling </li><li>Sanger WXS Variant Calling </li><li> GATK Mutect2 Variant Calling</li></ul> |
 
+## Filtered InDel Calls
+
+Data files containing the simple Insertions and Deletions (InDel) variants which were filtered through each caller's PASS tag and within only the open access regions.
```

Comment:
see the paragraph above this...the insertions and deletions isn't capitalized. Maybe this could be:

"Data files containing the simple insertions and deletions (InDel) variants which were filtered through each caller's PASS tag and located only within the open access regions"